### DISCUSSION TITLE
Items disappear into drawer controller

### DISCUSSION THREAD

**Discussion by Apromixately** [P3pp3rF1y/SophisticatedBackpacks#759] [Q&A]
I am using the "Run Inventory Interaction Upgrades" keybinding to insert items into a drawer controller. It seems to take anything from the backpack even if it doesn't exist in the drawer storage but destroy it in that case. Am I missing something or is that actually what's happening?

**Comment by P3pp3rF1y** [comment]
there was a bug like that several weeks ago make sure to update all sophisticated mods
